packages/legend-query-builder/src/components/result/tds/QueryBuilderTDSResultShared.tsx

```diff
@@ -436,39 +464,54 @@ export const QueryBuilderGridResultContextMenu = observer(
       );
     };
 
-    const handleCopyCellValue = applicationStore.guardUnhandledError(() =>
-      applicationStore.clipboardService.copyTextToClipboard(
-        data?.value?.toString() ?? '',
-      ),
-    );
+    const handleCopyCellValue = isEnterpriseModeEnabled
```

Comment:
lets not use `isEnterpriseModeEnabled` anywhere in this file, if we need to lets fork the whole component and move it to the corresponding file. we only keep it if  we need to share across both.